internal/devtools_browser.py

```diff
@@ -571,6 +571,16 @@ def collect_browser_metrics(self, task):
             accessibility_tree = None
             for name in self.job['customMetrics']:
                 custom_script = unicode(self.job['customMetrics'][name])
+                if custom_script.find('$WPT_URL') >= 0:
```

Comment:
Super nit here, maybe, but would $WPT_FINAL_URL or something like that make a bit more sense? @pmeenan

Reply:
Maybe $WPT_NAVIGATED_URL but it's not necessarily the final one, it is what was entered before any redirects may have kicked in. In the most basic case, it is the URL that was passed to WPT with the test (only becomes something different if there is a script with multiple navigate commands).

Reply:
Ah, gotcha. Yup, totally misread this last night.

Can we do $WPT_TEST_URL then?